$MirOS: ports/devel/apr/util/patches/patch-build_apr_common_m4,v 1.1 2006/10/05 21:11:39 bsiegert Exp $
--- build/apr_common.m4.orig	2007-05-29 11:01:39.000000000 +0200
+++ build/apr_common.m4	2008-09-21 22:04:49.000000000 +0200
@@ -797,18 +797,6 @@ dnl
 dnl APR_ENABLE_LAYOUT(default layout name [, extra vars])
 dnl
 AC_DEFUN(APR_ENABLE_LAYOUT,[
-AC_ARG_ENABLE(layout,
-[  --enable-layout=LAYOUT],[
-  LAYOUT=$enableval
-])
-
-if test -z "$LAYOUT"; then
-  LAYOUT="$1"
-fi
-APR_LAYOUT($srcdir/config.layout, $LAYOUT, $2)
-
-AC_MSG_CHECKING(for chosen layout)
-AC_MSG_RESULT($layout_name)
 ])
 
 
